tests/Unit/CssInlinerTest.php

```diff
@@ -2185,15 +2181,11 @@ public function inlineCssNotKeepsUnneededMediaRuleAfterEmptyMediaRule(string $em
     }
 
     /**
-     * @param string[] $precedingSelectorComponents Array of selectors to which each type of pseudo-component is
-     *                                              appended to create a selector for a CSS rule.
-     *                                              Keys are human-readable descriptions.
-     *
-     * @psalm-param array<string, string> $precedingSelectorComponents
-     *
-     * @return string[][]
+     * @param array<string, string> $precedingSelectorComponents selectors to which each type of pseudo-component is
+     *                                                           appended to create a selector for a CSS rule.
+     *                                                           Keys are human-readable descriptions.
```

Comment:
I think we should indent like this:

```php
/**
 * @param bool $needsExplanation This parameter requires some explanaining which spans onto
 *        multiple lines.
 * @param bool $selfExplanatory
 */
```

We have this style a fair bit (thought not entirely consistently) in the code base, and looking at other projects, it seems to be a happy middle ground.  (I doubt there's a sniff or suchlike for it, though you never know.)

Reply:
Fixed.